docs/QUICK_REFERENCE.md

```
@@ -0,0 +1,293 @@
+ *                 error:
+ *                   type: string
+ *                   example: "Validation failed"
+ */
+export async function POST(request: NextRequest) {
+  // Implementation
+}
+```
+
+## Standardized Tags
+
+### Admin
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)